misp/src/misp.py

```diff
@@ -488,8 +488,7 @@ def process_events(self, work_id, events):
                 bundle_objects.append(object_relationship)
 
             # Create the report if needed
-            # Report in STIX must have at least one object_refs
-            if self.misp_create_report and len(object_refs) > 0:
+            if self.misp_create_report:
```

Comment:
This will raise an error if object_refs length is 0, the STIX library does NOT allow empty report. We've some users having issues with the previous modification this is why we've to reintroduce it.

Reply:
My bad, those changes shouldn't have been committed. Did a rollback on the few unrelated changes.